pandas/conftest.py

```diff
@@ -97,3 +97,22 @@ def join_type(request):
     Fixture for trying all types of join operations
     """
     return request.param
+
+
+TIMEZONES = [None, 'UTC', 'US/Eastern', 'Asia/Tokyo', 'dateutil/US/Pacific']
+
+
+@pytest.fixture(params=TIMEZONES)
+def tz_naive(request):
```

Comment:
I wouldn't be averse to making the name of this slightly more explict

maybe ``tz_naive_fixture`` (maybe we should do this with the other fixtures as well, can be separate PR for that change)

Reply:
Done :)